### PR TITLE
Update accessible names for 'add profile' page buttons

### LINKED ISSUE #11156: [Settings> Add a new profile]: The Name property must not include the element's control type.
### Windows Terminal version (or Windows build number)

1.10.2383.0

### Other Software

**Test Environment:**
OS: Windows 11 Version 21H2 (OS Build 22000.168)
Screen Reader: Narrator
Tool: Accessibility Insights for Windows Version 1.1.1689.01

### Steps to reproduce

**Repro Steps:**

1. Open Windows Terminal.
2. Open Settings page using "Ctrl + ',' ".
3. Run Accessibility insights for Window on Settings Add a new Profile tab page and observe the issue.

**User Experience: -> Priority 3**
Screen Reader user are impacted here as control type is announced twice with control name which is confusing.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

**Attachments:**
[The Name property must not include the element's control type..zip](https://github.com/microsoft/terminal/files/7115865/The.Name.property.must.not.include.the.element.s.control.type.zip)

### Expected Behavior

The Name property must not include the element's control type. For 'New Empty Profile' Screen Reader should announce 'Create new Empty Profile button' and for 'Duplicate' Screen Reader should announce 'Duplicate button'. 

### Actual Behavior

The Name property has the element's control type. For 'New Empty Profile' button Screen Reader is announcing 'Create new button button' and For 'Duplicate' button Screen Reader is announcing 'Duplicate button button'. 

**Note Issue:** The Name must not include the same text as the LocalizedControlType.

### PR DESCRIPTION
**Pull request by carlos-zamora**
## Summary of the Pull Request
When using a screen reader, the buttons on the "add a new profile" page were being read weirdly:
- "New empty profile" button read as "create new button button"
- "duplicate" button read as "duplicate button button"

It's generally standard to read out the text inside the button, so I did just that by reusing the existing localized resources. This also removes the redundant "button" that is said by the screen reader.

I also removed the unused `AutomationId` and unnecessary `Button.Content` tags. 

#11156 can be closed upon validation by the accessibility team.

## Validation Steps Performed
✅ navigate to both buttons using Narrator; make sure it sounds right

### CODE REVIEW

**Comment by carlos-zamora** [comment]
@DHowett I'm adding the `zStable-Service-Consider` tag because it's low-hanging fruit. But feel free to remove it if you disagree.

Also, I'd like to remove...
```
<data name="AddProfile_AddNewButton.Tag" xml:space="preserve">
    <value>Create New Button</value>
  </data>
  <data name="AddProfile_DuplicateButton.Tag" xml:space="preserve">
    <value>Duplicate Button</value>
  </data>
```
from the resw file, since now nobody is referencing it. Idk if we have a process for doing that? Should I file that or just ignore it and move on with life?

**Comment by DHowett** [comment]
> @DHowett Dustin Howett FTE I'm adding the `zStable-Service-Consider` tag because it's low-hanging fruit. But feel free to remove it if you disagree.
> 
we're not updating 1.11 in-market; all users who are eligible for it are eligible for 1.12!

File: src/cascadia/TerminalSettingsEditor/AddProfile.xaml [hunk 1]
```diff
@@ -68,21 +64,17 @@
                         </muxc:RadioButtons.ItemTemplate>
                     </muxc:RadioButtons>
                 </local:SettingContainer>
-                <Button x:Uid="AddProfile_DuplicateButton"
+                <Button x:Name="DuplicateButton"
                         Margin="{StaticResource StandardControlMargin}"
-                        AutomationProperties.AutomationId="AddProfile_DuplicateButton"
-                        AutomationProperties.Name="{Binding Tag, RelativeSource={RelativeSource Self}}"
```
Comment (zadjii-msft) [inline src/cascadia/TerminalSettingsEditor/AddProfile.xaml:74]: > `"{Binding Tag, RelativeSource={RelativeSource Self}}"`

wait is that some XAML magic for "use this other property on myself"? That's really neat!

**Review by zadjii-msft — APPROVED** [review]

File: src/cascadia/TerminalSettingsEditor/AddProfile.xaml [hunk 2]
```diff
@@ -23,19 +23,15 @@
 
     <ScrollViewer ViewChanging="ViewChanging">
         <StackPanel Style="{StaticResource SettingsStackStyle}">
-            <Button x:Uid="AddProfile_AddNewButton"
-                    AutomationProperties.AutomationId="AddProfile_AddNewButton"
-                    AutomationProperties.Name="{Binding Tag, RelativeSource={RelativeSource Self}}"
+            <Button x:Name="AddNewButton"
                     Click="AddNewClick"
                     Style="{StaticResource AccentButtonStyle}">
-                <Button.Content>
```
Comment (DHowett) [inline src/cascadia/TerminalSettingsEditor/AddProfile.xaml:31]: sorry to be a pill, but can you not do this part if it is not part of the bug fix? It's harder to service bigger diffs... and you're going to conflict with #12326 as it is and if we can make the actual bug fix part more obvious we should

**Review by DHowett — CHANGES_REQUESTED** [review]
as noted

**Review by DHowett — APPROVED** [review]

**Pull request merged**